compiler/frontend/src/org/jetbrains/kotlin/resolve/calls/tower/NewResolutionOldInference.kt

```diff
@@ -388,7 +396,15 @@ class NewResolutionOldInference(
             dispatchReceiver: ReceiverValueWithSmartCastInfo?,
             extensionReceiver: ReceiverValueWithSmartCastInfo?
         ): Collection<FunctionDescriptor> {
-            return candidateInterceptor.interceptFunctionCandidates(initialResults, this, resolutionContext, resolutionScope, callResolver, name, location)
+            return candidateInterceptor.interceptFunctionCandidates(
```

Comment:
Please, separate plain formatting changes to different commits when it's not very hard